src/Camera.cpp

```diff
@@ -11,26 +11,26 @@
 Camera::Camera( glm::vec3 p, glm::vec3 f, glm::vec3 u ):
     pos( p )
     , forward( f )
-    , up( u ) {
+    , up( u )
+    , fov( 45 ) {
 
     ViewMatrix = glm::lookAt (
                      pos, // Camera position in World Space
-                     forward, // and looks at the origin
+                     forward - pos, // and looks at the origin
                      up  // Head is up (set to 0,-1,0 to look upside-down)
                  );
 
     //perspective (T const &fovy, T const &aspect, T const &near, T const &far)
-    float initialFoV = 45.0f;
-    float near = 0.1f;
+    float near = 1.0f;
     float far = 100.f;
     float aspect = 1.0;
-    ProjectionMatrix = glm::perspective( glm::radians( initialFoV ), aspect, near, far );
+    ProjectionMatrix = glm::perspective( glm::radians( fov ), aspect, near, far );
 }
 
 void Camera::UpdateViewMatrix() {
     ViewMatrix = glm::lookAt (
                      pos,
-                     forward,
+                     forward - pos,
```

Comment:
I'm not sure why you're subtracting pos from forward.

If forward = glm::vec3(0.f, 0.f, 1.f) then camera is always going to point to glm::vec3(0.f, 0.f, 1.f). I'm guessing you want to use forward as a direction? If so, it should be `pos + forward`